documentation/modules/cnv-cnv-live-prerequisites.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="cnv-cnv-live-prerequisites_{context}"]
+= {virt} live migration prerequisites
+
+[role="_abstract"]
+Live migration between {virt} clusters using {project-first} has its own set of prerequisites:
+
+* {project-short} 2.10.0 or later installed. For more information, see xref:upgrading-mtv-ui_mtv[Upgrading the {project-full}]. {project-short} treats all {virt} migrations run on {project-short} 2.9 or earlier as cold migrations, even if they are configured as live migrations.
+* {virt} 4.20.0 or later installed on both source and target clusters.
+* In the `KubeVirt` resource of both clusters in the `featureGates` of the YAML,`DecntralizedLiveMigration` is listed. You must have `cluster-admin` privileges to set this field.
+* Connectivity between the clusters must be established, including connectivity for state transfer. Technologies such as Submariner can be used for this purpose.
+* `forklift-controller` has access to read and create `VirtualMachineInstanceMigration` instances on both the source and target clusters.
+* The target cluster has `VirtualMachineInstanceTypes` and `VirtualMachinePreferences` that match those used by the VMs on the source cluster.
+
```

Comment:
Do we also need to set the feature flag on MTV side I assume `feature_ocp_live_migration`? since it's a tech preview feature
cc @mnecas

Reply:
@mansam  @mnecas @avivtur: I previously had the following as a prerequisite. Sam said it should be removed. Aviv thinks we need it since the feature is a Tech Preview. Can I have a group decision on this? Thanks. 

 
* In the {project-short} Operator, in the `spec` portion of the `forklift-controller` YAML, `feature_ocp_live_migration` is set to `true`. You must have `admin-cluster` privileges to set this label.